metacatalog/ext/standards_export/schemas/iso19115/iso19115-2.j2

```diff
@@ -283,6 +274,11 @@
                     {% endfor %}
                 </gmd:EX_Extent>
             </gmd:extent>
+            {% for detail in details %}
+            <gmd:supplementalInformation>
+                <gco:CharacterString>{{detail}}</gco:CharacterString>
+            </gmd:supplementalInformation>
```

Comment:
Does this actually work for all kinds of details?

How is the information organized in the ISO then? We have way more that just plain character-strings, right?

Also, I am not sure if the supplementalInformation is the best tag. We are more after an extension of the schema itself, right?

Maybe https://schemas.isotc211.org/19115/-1/mex/1.3.0/ is more appropriate to include the logic represented by our Detail class.

Reply:
With the implementation of `details` as `<supplementalInformation>`, details are just printed as strings of key-value pairs:

For test data:
![image](https://user-images.githubusercontent.com/23619690/221926167-061e9710-5cda-47a0-a17b-3fea71c8a84d.png)

For 'actual' data:
![image](https://user-images.githubusercontent.com/23619690/221926276-993f128e-a880-4329-a2b7-f9c0dcd1ebe7.png)

I think putting details into `<supplementalInformation>` is appropriate according to the official definition ("Other descriptive information about the dataset."). But a problem i just discovered is, that `<supplementalInformation>` is not repeatable.

Reply:
I just tried to understand https://schemas.isotc211.org/19115/-1/mex/1.3.0/ but I didn't really get how the xml file and tags would look like for our details, an xml example would be great but i could not find one.